### DISCUSSION TITLE
How to use where in update func with relation field

### DISCUSSION THREAD

**Discussion by IRediTOTO** [prisma/prisma#12413] [Q&A] [answered]
Hi, I need update a post which connect to a user.
When someone request to update a field in that post, what is the best way to check that is user is the post's owner?

```javascript
update(user: JWT, postId: number, updatePostDto: UpdatePostDto) {
    return this.prm.deal.update({
      where: { id:postID, authorId: user.id }, // I have user ID here
      data: updateDealDto,
    });
  }
```
I don't really want use `updateMany`. It doesn't return updated post.

**Comment by austincrim** [comment, marked as answer]
Hey @IRediTOTO 👋 ,

The easiest way to do this is to update the post *through* the user.

```ts
let result = await prisma.user.update({
  where: { id: userId },
  data: {
    posts: {
      update: {
        where: { id: postId },
        data: { postDTO }
      }
    }
  }
})
```

Let me know if that will work for you!

**Reply by IRediTOTO** [reply]
Thank you so much :D Great idea. But I still wish have a clear solution in future

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
